README.md

```diff
@@ -1,41 +1,95 @@
 
 ```bash
-$ groups
+sudo usermod -aG docker $USER
 ```
 
-If you dont see `docker`, then add your user to the Docker group:
+### Binary Packages
+
+`launch_ros_sandbox` is not yet available as a binary package using APT or
+any other method.
+
+### Installing from source
+
+#### Dashing (`dashing-devel` branch)
+
+This is the recommended way to install this software.
+
+* Install ROS 2 Dashing on your machine following the
+  [official instructions][ros2_dashing_setup]. We recommend you use the
+  official binary packages for your platform, if available.
+* Checkout the code source and compile it as follow:
 
 ```bash
-$ sudo usermod -aG docker $USER
+# If you use bash or zsh, source setup.bash or setup.zsh, instead of setup.sh
+source /opt/ros/dashing/setup.sh
+mkdir -p ~/ros2_dashing_ros_launch_sandbox_ws/src
+cd ros2_dashing_ros_launch_sandbox_ws
+# Clone this package repository using vcs.
+curl https://raw.githubusercontent.com/aws-robotics/launch-ros-sandbox/master/launch_ros_sandbox.dashing.repos | vcs import src/
+# Use colcon to compile launch_ros_sandbox code and all its dependencies
+colcon build --packages-up-to launch_ros_sandbox
+```
+
+#### Latest (unstable development - `master` branch)
+
+Please follow those instructions if you plan to contribute to this repository.
+
+* Install all software dependencies required for ROS 2 development by
+  following the [ROS 2 documentation][ros2_latest_setup].
+* Checkout the code source and compile it as follow:
+
+```bash
+mkdir -p ~/ros2_latest_ros_launch_sandbox_ws/src
+cd ros2_latest_ros_launch_sandbox_ws
+# Use vcs to clone all required repositories
+curl https://raw.githubusercontent.com/ros2/ros2/dashing/ros2.repos | vcs import src/
+curl https://raw.githubusercontent.com/aws-robotics/launch-ros-sandbox/master/launch_ros_sandbox.repos | vcs import src/
+# Use colcon to compile launch_ros_sandbox code and all its dependencies
+colcon build --packages-up-to launch_ros_sandbox
 ```
 
 ## Usage
 
-A working example is provided in [examples/minimal_sandboxed_node_container.launch.py](https://github.com/aws-robotics/launch-ros-sandbox/tree/master/examples/minimal_sandboxed_node_container.launch.py).
+A working example is provided in
+[examples/minimal_sandboxed_node_container.launch.py][ex_minimal_sandboxed_node_container_launch]
 
 ```bash
-$ ./examples/minimal_sandboxed_node_container.py
+./examples/minimal_sandboxed_node_container.py
 ```
 
 Creating a sandboxed node is very similar to creating a regular launch file.
 
-Add a `SandboxedNodeContainer()` action like you would with a regular launch file, but make sure to provide the `sandbox_name` and `policy`.
-Adding nodes is also similar to regular launch files, however, you should use `launch_ros_sandbox.descriptions.SandboxedNode()` instead.
+Add a `SandboxedNodeContainer()` action like you would with a regular launch
+file, but make sure to provide the `sandbox_name` and `policy`.
```

Comment:
Should we stick to the convention of keeping  sentences on one line?

Reply:
We should!

Reply:
We should not have lines longer than 80 cols (or whatever limit we want to adopt).